keyboards/iris/keymaps/dbroqua/rules.mk

```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = no
+BACKLIGHT_ENABLE = no
+
+ifndef QUANTUM_DIR
```

Comment:
This ifndef block is not needed.  It's a hold over from the old makefile structure.

Reply:
ok